specs/regression.txt

```diff
@@ -747,3 +747,54 @@ ISSUE 421
 
 <h1>assimp-rs <a href="https://crates.io/crates/assimp"><img alt="" src="http://meritbadge.herokuapp.com/assimp"></a></h1>
 ````````````````````````````````
+
+ISSUE 443 (note the spaces on the blank lines)
+
+```````````````````````````````` example
+* A list.
+
+   * A sublist.
+
+   * Another sublist.
+
+
+* A list.
+ 
```

Comment:
It's unfortunate that GitHub doesn't show whitespace on blank lines here. It makes this test look a little silly ^^